library/src/blas3/rocblas_trmm_strided_batched.cpp

```diff
@@ -0,0 +1,373 @@
+                                                     rocblas_stride    stride_a,
+                                                     T*                c,
+                                                     rocblas_int       ldc,
+                                                     rocblas_stride    stride_c,
+                                                     rocblas_int       batch_count)
+    {
+        if(!handle)
+            return rocblas_status_invalid_handle;
+
+        auto layer_mode = handle->layer_mode;
+        if(layer_mode
+               & (rocblas_layer_mode_log_trace | rocblas_layer_mode_log_bench
+                  | rocblas_layer_mode_log_profile)
+           && (!handle->is_device_memory_size_query()))
+        {
+            auto side_letter   = rocblas_side_letter(side);
+            auto uplo_letter   = rocblas_fill_letter(uplo);
+            auto transa_letter = rocblas_transpose_letter(transa);
+            auto diag_letter   = rocblas_diag_letter(diag);
+
+            if(handle->pointer_mode == rocblas_pointer_mode_host)
+            {
+                if(layer_mode & rocblas_layer_mode_log_trace)
+                    log_trace(handle,
+                              rocblas_trmm_strided_batched_name<T>,
+                              side,
+                              uplo,
+                              transa,
+                              diag,
+                              m,
+                              n,
+                              log_trace_scalar_value(alpha),
+                              a,
+                              lda,
+                              stride_a,
+                              c,
+                              ldc,
+                              stride_c,
+                              batch_count);
+
+                if(layer_mode & rocblas_layer_mode_log_bench)
+                {
+                    log_bench(handle,
+                              "./rocblas-bench -f trmm_strided_batched -r",
+                              rocblas_precision_string<T>,
+                              "--side",
+                              side_letter,
+                              "--uplo",
+                              uplo_letter,
+                              "--transposeA",
+                              transa_letter,
+                              "--diag",
+                              diag_letter,
+                              "-m",
+                              m,
+                              "-n",
+                              n,
+                              LOG_BENCH_SCALAR_VALUE(alpha),
+                              "--lda",
+                              lda,
+                              "--stride_a",
+                              stride_a,
+                              "--ldb",
+                              ldc,
+                              "--stride_b",
+                              stride_c,
```

Comment:
--stride_b but the interface is using stride_c.

Reply:
Changed to consistently use b, ldb, stride_b